### DISCUSSION TITLE
Sending value of one claim as another

### DISCUSSION THREAD

**Discussion by jaredmarcuccilli** [kanidm/kanidm#3166] [Q&A] [answered]
Hello!

I have a particular application (Cloudflare Zero Trust) that expects a user's display name to be sent as "username." The field that uniquely identifies a user can be customized, but the display name cannot - it must be "username."

Is it possible to send "displayname" from Kanidm as "username" to the application?

**Comment by Firstyear** [comment, marked as answer]
Currently this isn't possible. There has been other requests for claim templating that we haven't decided what to do with yet.

This sounds like a bug in cloudflare though.

**Reply by jaredmarcuccilli** [reply]
Yes I would agree that Cloudflare should allow you to specify which field is used as the display name. Fortunately it is only cosmetic. Thank you for the great software!
